components/slider/src/slider.types.ts

```diff
@@ -39,6 +44,7 @@ export type RangeSliderProps =
   & {
     disabled?: boolean;
     marks?: boolean | MarkDef[];
+    sectionLabels?: boolean | SectionLabelDef[]; // why ever boolean??
```

Comment:
It doesn't have to be boolean. Marks are typed as boolean since if you set "marks": true together with "steps", you will get tiny marks on the track that highlights the step values. See the "With steps" example.

Reply:
Aha, thank you for the explanation!